auth/auth.go

```diff
@@ -42,11 +43,20 @@ type userByEmailInfo struct {
 // Creates a new Authenticator that stores user info in the given Bucket.
 func NewAuthenticator(bucket base.Bucket, channelComputer ChannelComputer) *Authenticator {
 	return &Authenticator{
-		bucket:          bucket,
-		channelComputer: channelComputer,
+		bucket:            bucket,
+		channelComputer:   channelComputer,
+		sessionCookieName: DefaultCookieName,
```

Comment:
I was debating whether to add `sessionCookieName` to the param list, but it meant changing a lot more code that would essentially be noise in most of the calling code (99% tests).  Any thoughts/preferences here?

Reply:
As long as we're using dbContext.Authenticator() to retrieve an authenticator everywhere (and it looks like we are), this should be fine.